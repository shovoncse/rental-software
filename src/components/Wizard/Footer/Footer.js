const Footer = ({credit}) => {
    return (
        <>
            <div className="flex justify-center px-10">
                <p className="">Made with ❤️ by {credit}</p>
            </div>
        </>
    )
}

export default Footer