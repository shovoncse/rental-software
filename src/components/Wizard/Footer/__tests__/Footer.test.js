import React from 'react';
import ReactDOM from 'react-dom';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import Footer from '../Footer';

afterEach(cleanup);

describe('<Footer />', () => {
  it('should render without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Footer></Footer>, div);
  });

  it('should render an <div> tag', () => {
    const {
      container: { firstChild },
    } = render(<Footer />);
    expect(firstChild.tagName).toEqual('DIV');
  });

  it('should have a class attribute', () => {
    const {
      container: { firstChild },
    } = render(<Footer />);
    expect(firstChild.hasAttribute('class')).toBe(true);
  });

  it('matches snapshot', () => {
    const tree = renderer.create(<Footer name="Footer"></Footer>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
