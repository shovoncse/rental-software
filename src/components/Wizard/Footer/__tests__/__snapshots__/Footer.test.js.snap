// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> matches snapshot 1`] = `
<div
  className="flex justify-center px-10"
>
  <p
    className=""
  >
    Made with ❤️ by 
  </p>
</div>
`;
